articles/azure-cache-for-redis/cache-best-practices.md

```diff
@@ -47,7 +47,7 @@ There are several things related to memory usage within your Redis server instan
 ## Client library specific guidance
  * [StackExchange.Redis (.NET)](https://gist.github.com/JonCole/925630df72be1351b21440625ff2671f#file-redis-bestpractices-stackexchange-redis-md)
  * [Java - Which client should I use?](https://gist.github.com/warrenzhu25/1beb02a09b6afd41dff2c27c53918ce7#file-azure-redis-java-best-practices-md)
- * [Lettuce (Java)](https://gist.github.com/warrenzhu25/181ccac7fa70411f7eb72aff23aa8a6a#file-azure-redis-lettuce-best-practices-md)
+ * [Lettuce (Java)](https://github.com/Azure/AzureCacheForRedis/blob/main/Lettuce%20Best%20Practices.md)
```

Comment:
The new link seems to be just a simple "getting started" topic, while the old gist link had some detailed best practices. Why the switch?

Reply:
We will need to vet things from the old gist and then put only those into the new gist which are accurate.